src/helpers/utilities.ts

```diff
@@ -451,6 +451,39 @@ export const omitFlatten = <T extends object, K extends keyof T>(
   return newObj;
 };
 
+/**
+
+ * @desc Converts the first character of string to upper case and the remaining to lower case
+ */
+export const capitalize = (string?: string | undefined) => {
+  if (!string) return '';
+  return string
+    ? string.charAt(0).toUpperCase() + string.slice(1).toLowerCase()
+    : '';
+};
+
+export const notEmpty = <TValue>(
```

Comment:
Can we add JSDOC style comments like the rest of the functions to these ones?

Reply:
sure